### PR TITLE
added Pie, vertical Bar and Composed Charts

### PR DESCRIPTION
**Pull request by naresh130199**
Added graph components

### CODE REVIEW

File: dashboard/src/components/Charts/BarChart/VerticalBarChart.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  Bar,
+  XAxis,
+  YAxis,
+  Tooltip,
+  ResponsiveContainer,
+  CartesianGrid,
+} from 'recharts';
+export const VerticalBarChart = ({ data }) => {
+  // data = array of objects and 
+  // BarChartData for Bar chart component, XAxisValue and YAxisValue
```
Comment (omni-srejitk) [inline dashboard/src/components/Charts/BarChart/VerticalBarChart.jsx:13]: Where do we take in the XAxis Value and YAxis Value? Did we miss that here

File: dashboard/src/components/Charts/ComposedChart/ComposedChartComp.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            fontFamily: 'Inter',
+          }}
+        />
+        <Tooltip 
+          contentStyle={{ backgroundColor: "black", opacity: '0.9', borderRadius: '1rem',}}
+          wrapperStyle={{outline: 'none'}}
+          labelStyle={{ color: "white" }}
+          itemStyle={{ color: "cyan" }}
+        />        
+        {/* <Area type="monotone" dataKey="" fill="#8884d8" stroke="#8884d8" /> */}
```
Comment (omni-srejitk) [inline dashboard/src/components/Charts/ComposedChart/ComposedChartComp.jsx:49]: Remove the commented code if not needed

File: dashboard/src/components/Charts/PieChart/PieChartComp.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    height={'100%'}
+    margin={{ top: 0 }}
+    >
+      <PieChart 
+      width={'100%'} 
+      height={'100%'}
+      >
+        <Pie 
+        data={data} 
+        dataKey="value"
```
Comment (omni-srejitk) [inline dashboard/src/components/Charts/PieChart/PieChartComp.jsx:26]: We need to take in the dataKey as a prop here too

File: dashboard/src/components/Charts/BarChart/VerticalBarChart.jsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  return (
+    <ResponsiveContainer width={'100%'} height={'100%'}>
+      <BarChart 
+      width={'100%'} 
+      height={'100%'} 
+      data={data} 
+      layout="vertical"
+      >
+        <XAxis
+          dataKey={'XAxisValue'}
```
Comment (omni-srejitk) [inline dashboard/src/components/Charts/BarChart/VerticalBarChart.jsx:24]: Correct this. This should take in the prop that you add rather than the string

**Review by omni-srejitk — CHANGES_REQUESTED** [review]
Good work. Left few minor changes. 
Keep the props in mind, and in the arguments section

File: dashboard/src/components/Charts/BarChart/VerticalBarChart.jsx (hunk 1 above)
Comment (naresh130199) [reply dashboard/src/components/Charts/BarChart/VerticalBarChart.jsx]: changed that now data props should provide the required data

File: dashboard/src/components/Charts/ComposedChart/ComposedChartComp.jsx (hunk 2 above)
Comment (naresh130199) [reply dashboard/src/components/Charts/ComposedChart/ComposedChartComp.jsx]: yep removed it

File: dashboard/src/components/Charts/PieChart/PieChartComp.jsx (hunk 3 above)
Comment (naresh130199) [reply dashboard/src/components/Charts/PieChart/PieChartComp.jsx]: done

File: dashboard/src/components/Charts/BarChart/VerticalBarChart.jsx (hunk 4 above)
Comment (naresh130199) [reply dashboard/src/components/Charts/BarChart/VerticalBarChart.jsx]: yeah, now it will be taking in props

**Pull request merged**